clang/include/clang/Basic/DiagnosticDriverKinds.td

```diff
@@ -61,6 +61,10 @@ def err_drv_no_cuda_libdevice : Error<
   "via '--cuda-path', or pass '-nocudalib' to build without linking with "
   "libdevice">;
 
+def warn_function_fp_accuray_already_set : Warning <"FP accuracy value of '%0' has already "
+  "been assigned to function '%1'">;
+def warn_all_fp_accuray_already_set : Warning <"FP accuracy value of '%0' has already "
+  "been assigned to all functions in the program">;
```

Comment:
Should there be a way to disable these diagnostics?

Reply:
It is an informative diagnostic, but typically warnings have a way to be disabled.

Reply:
I can create the warning in a new group:
def FPAccuracyWrongValue  : DiagGroup<"wrong-value-of-fp-accuracy">;
and then use the -Wno-wrong-value-of-fp-accuracy on the command line to disable the warning. Is that what you mean?

Reply:
Yup, that's what I mean.  Maybe:  "fp-accuracy-value" would be sufficient here for the name.